lib/plugin.js

```diff
@@ -7,16 +7,23 @@ window.io = require('socket.io-client')
 <% } %>
 
 function getHeaders ({ app }) {
-  const headers = {}
+  const headers = {
+    referer: location.origin
+  }
 
   <% if (options.authModule) { %>
   if (app.$auth) {
     const strategy = app.$auth.strategy
-    const tokenName = strategy.options.tokenName || 'Authorization'
-    const token = app.$auth.getToken(strategy.name)
 
-    if (token) {
-      headers[tokenName] = token
+    if (strategy.options.name === 'laravelSanctum') {
+      headers['X-XSRF-TOKEN'] = app.$auth.$storage.getCookies()['XSRF-TOKEN']
+    } else {
+      const tokenName = strategy.options.token.name || 'Authorization'
+      const token = app.$auth.token.get()
```

Comment:
This doesn't seem to work with the latest nuxt/auth version. Changing the line to `const token = strategy.token.get()` fixes this PR.

Reply:
Yes, you're right. The token instance has been moved to strategy.

Reply:
Fixed in 8635bd52e128fe72f03f7f67d4e46e4649bd09fe